backend/routes.py

```
@@ -828,9 +1255,279 @@ def run_manual_cleanup(dry_run: bool = True):
+            "pending_cleanup": pending_cleanup,
+        }
+
+    except Exception as e:
+        raise HTTPException(
+            status_code=500, detail=f"Failed to get temp storage stats: {str(e)}"
+        ) from e
+
+
+@router.post("/temp-storage/cleanup")
+def cleanup_temp_storage(
+    data: dict = {}, authenticated: bool = Depends(verify_api_key)
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)